packages/manager/modules/exchange/src/domain/dkim-autoconfig/domain-dkim-autoconfig.controller.js

```diff
@@ -1,6 +1,8 @@
 import { DKIM_CONFIGURATION_GUIDE } from './domain-dkim-autoconfig.constants';
 
-export default class ExchangeDomainDkimAutoconfigCtrl {
+import DkimAutoConfigurator from './dkim-auto-configurator';
+
+export default class ExchangeDomainDkimAutoconfigCtrl extends DkimAutoConfigurator {
```

Comment:
Why do you split your controller in 2 classes whereas you don't use it anywhere else ?

Reply:
Same